### DISCUSSION TITLE
Some Sensor - Unknown Result

### DISCUSSION THREAD

**Discussion by x230-thinkpad** [txubelaxu/esphome-jk-bms#30] [Q&A]
hi @txubelaxu 
thank you very much for you do with JK-PB Series BMS

I have a little problem, some sensor always become unknown 

sensor:
    battery_type:
      name: "${bms0_name} battery type"
    powertube_temperature_protection:
      name: "${bms6_name} powertube temperature protection" 
    powertube_temperature_protection_recovery:
      name: "${bms6_name} powertube temperature protection recovery" 
	  
text_sensor:
    operation_status:
      name: "${bms0_name} operation status"

![image](https://github.com/user-attachments/assets/fdb6f5dd-79b9-4224-b6c5-db0392ff2d3b)
![image](https://github.com/user-attachments/assets/df39f783-b823-47bd-a90d-a4379ef5db92)

I used esp32-S3-example-jkpb-rs485_0master_7slaves.yaml and all is good, 1 ESP can handle 7 JK bms very well
![image](https://github.com/user-attachments/assets/6ac01c6c-dd64-439a-ba01-b822b80589d0)

**Comment by txubelaxu** [comment]
Sorry of the delay. Is it now solved?

**Reply by x230-thinkpad** [reply]
> Sorry of the delay. Is it now solved?

yes, it still unknown result for 4 parameters

**Reply by txubelaxu** [reply]
Well, I have reviewed first one and it is not present at the source information. Others I think there were renamed. Could you take a look to esp32-example-jkpb-rs485_0master_2slaves.yaml file. I think there are all the params available.
